File: JavaScript/exercicio17.js
function media(){
    let nota1 = Number(prompt("Digite a primeira nota:"))
    let nota2 = Number(prompt("Digite a segunda nota:"))

    let media = (nota1 + nota2) / 2

    alert("A média das duas notas é: "+media)

    document.getElementById("mostra").innerHTML = "Nota 1: "+nota1
    document.getElementById("mostra1").innerHTML = "Nota 2: "+nota2
    document.getElementById("mostra2").innerHTML = "Media: "+media

    if(media > 6){
        alert("Aluno aprovado")
       document.getElementById("mostra3").innerHTML = "Aluno aprovado" 
    } else{
        alert("Aluno reprovado")
        document.getElementById("mostra3").innerHTML = "Aluno reprovado"
    }

    document.querySelector("#btn").innerHTML = "Calcular outra média"
    
}

File: Java/faccat/Exercicio17.java
package faccat;

import java.util.Scanner;

public class Exercicio17 {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int nota1, nota2, media;
		
		System.out.println("Digite a primeira nota");
		nota1 = sc.nextInt();
		
		System.out.println("Digite a outra nota");
		nota2 = sc.nextInt();
		
		media = (nota1 + nota2) / 2;
		System.out.println("A média das notas digitadas é "+media);
		
		if (media >= 6) {
			System.out.println("Aluno aprovado!");
		} else {
			System.out.println("Aluno reprovado!");
		}
		
		sc.close();
	}
}
